### PR TITLE
feat: Add support for RJSF

### PR DESCRIPTION
**Pull request by eshankvaish**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
     "name": "@devtron-labs/devtron-fe-common-lib",
-    "version": "v0.0.32",
+    "version": "0.0.39-beta-1",
```
Comment (eshankvaish) [inline package-lock.json:3]: Actions were failing as we are using npm ci which requires the lock file to list the dependencies to be installed.

File: src/Common/RJSF/common/FieldRow.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    shouldAlignCenter = true,
+}: FieldRowProps) => (
+    <div
+        className={
+            showLabel
+                ? `display-grid dc__gap-12 rjsf-form-template__field ${shouldAlignCenter ? 'flex-align-center' : ''}`
+                : ''
+        }
+    >
+        {showLabel && (
```
Comment (vivek-devtron) [inline src/Common/RJSF/common/FieldRow.tsx:19]: We can add the label check here as well

File: src/Common/RJSF/templates/ArrayFieldItemTemplate.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    registry,
+    uiSchema,
+}: ArrayFieldTemplateItemType) => {
+    const { RemoveButton } = registry.templates.ButtonTemplates
+
+    return (
+        <div className="display-grid rjsf-form-template__array-field-item flex-align-center dc__gap-8 mb-12">
+            {children}
+            {hasToolbar && hasRemove && (
+                <div>
```
Comment (vivek-devtron) [inline src/Common/RJSF/templates/ArrayFieldItemTemplate.tsx:22]: Is this required? can't we remove this?

File: src/Common/RJSF/common/FieldRow.tsx (hunk 2 above)
Comment (eshankvaish) [reply src/Common/RJSF/common/FieldRow.tsx:19]: Added a comment for better readability

Comment (eshankvaish) [reply src/Common/RJSF/common/FieldRow.tsx:19]: It is required for the grid layout. Added a comment for better readability

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**